packages/libs/src/sdk/services/Auth/types.ts

```diff
@@ -24,4 +26,38 @@ export type AuthService = {
    * Get the sender address, used for EntityManager writes
    */
   getAddress: () => Promise<string>
+
+  signUp?: ({
```

Comment:
These are all Hedgehog specific, and should be part of HedgehogAuth's type but not the others

Reply:
This only works if we do the above suggested approach, which I guess is nice.... and maybe a +1 to that approach... But otherwise, these need to be on the exported AuthService in order for you to call into it